internal/plugins/helm/plugin.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helm
+
+import (
+	"sigs.k8s.io/kubebuilder/pkg/plugin"
+)
+
+const pluginName = "helm" + plugin.DefaultNameQualifier
+
+var (
+	_ plugin.CreateAPIPluginGetter = Plugin{}
+)
+
+type Plugin struct {
+	createAPIPlugin
+}
+
+func (Plugin) Name() string                           { return pluginName }
+func (Plugin) Version() plugin.Version                { return plugin.Version{} }
+func (Plugin) SupportedProjectVersions() []string     { return nil }
```

Comment:
The helm plugin itself should be added in another PR where we decide on a version, supported project versions, and name.

Reply:
If we go that route, we would need to export the `initPlugin` struct so that we could access it directly in the CLI.

I don't have a strong opinion on this though since `SupportedProjectVersions()` returning `nil` should mean this can't actually be used as is in a CLI (IIUC).

For `Name()`, we could return `""` for now.

Reply:
Hi @estroz, 

The strategy is:
- Add only the plugins(without the config) 
- Then, when we have to init and create api for Ansible and Helm we will do 1 pr to call then in the new command.
- So, after that I see we following up with; the parse of the flags (domain/group/version), config and then cli. 

In this way, the version will be just decided when we set up the config which is out of the scope. I hope that it clarifies.

Reply:
Ideally `Plugin` should be added in a separate PR and a `createAPIPlugin` field added here to keep the scope tight.

However you're right, my comment on name etc. shouldn't block this PR, since we should be deciding these defining attributes when we add the plugin to the new CLI.

Reply:
Hi @estroz, we have the init and api. We will merge the init first and then, rebase this one with the master and it will be just the api as you suggested. See; https://github.com/operator-framework/operator-sdk/pull/3239.

I am looking for your ok/lgtm for we are able to move forward with it. ( init first -> rebase -> then just api )

Reply:
That's fine I suppose.